LEGO1/legoentity.cpp

```diff
@@ -0,0 +1,107 @@
+#include "legoentity.h"
+
+// OFFSET: LEGO1 0x100105f0 STUB
+LegoEntity::LegoEntity()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c290
+LegoEntity::~LegoEntity()
+{
+  Destroy();
+}
+
+// OFFSET: LEGO1 0x100114f0 STUB
+long LegoEntity::Notify(MxParam &p)
+{
+  // TODO
+
+  return 0;
+}
+
+// OFFSET: LEGO1 0x100107e0 STUB
+undefined4 LegoEntity::VTable0x18(undefined4 param)
+{
+  // TODO
+
+  return undefined4();
+}
+
+// OFFSET: LEGO1 0x10010810 STUB
+void LegoEntity::Destroy()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x10010e10 STUB
+void LegoEntity::VTable0x20(char *param)
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x100108a0 STUB
+void LegoEntity::VTable0x24(undefined4 param_1, undefined1 param_2, undefined1 param_3)
+{
+  // TODO
+}
```

Comment:
I don't think this one is correct.
It's also present in a vtable at `100d487c`.